core/txpool/blobpool/blobpool.go

```diff
@@ -1241,6 +1242,14 @@ func (p *BlobPool) add(tx *types.Transaction) (err error) {
 		addtimeHist.Update(time.Since(start).Nanoseconds())
 	}(time.Now())
 
+	// When the guardian module is enabled, check if the sender or recipient in the
+	// transaction is in the filter file
+	if instance, err := guardian.GetInstance(); err == nil {
```

Comment:
no need for filtering blob transactions as eip4844 is not enabled in story right now

Reply:
Done